packages/core/src/utils/ConstantsUtil.ts

```diff
@@ -61,7 +61,99 @@ export const ConstantsUtil = {
     Base: 'base'
   },
 
-  WC_COINBASE_ONRAMP_APP_ID: 'bf18c88d-495a-463b-b249-0b9d3656cf5e'
+  WC_COINBASE_ONRAMP_APP_ID: 'bf18c88d-495a-463b-b249-0b9d3656cf5e',
+
+  SUGGESTED_TOKENS: [
```

Comment:
These are predefined and temporary values, we should select which tokens to show first